test/extended/authorization/rbac/groups_default_rules.go

```diff
@@ -182,47 +183,60 @@ var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] The default clust
 
 	oc := exutil.NewCLI("default-rbac-policy")
 
-	g.It("should have correct RBAC rules", func() {
-		kubeInformers := informers.NewSharedInformerFactory(oc.AdminKubeClient(), 20*time.Minute)
-		ruleResolver := exutil.NewRuleResolver(kubeInformers.Rbac().V1()) // signal what informers we want to use early
+	g.It(
+		fmt.Sprintf("should have correct RBAC rules [apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s]",
```

Comment:
This is quite complex. `allUnauthenticatedRules` is the source of so many groups. I'd rather turn `allUnauthenticatedRules` into a api-group-key map and loop over all apigroups over `g.It("should have correct RBAC rules", func() {` rather than have all groups in a single test name.

Reply:
Mind if we change this in the next PR?

Reply:
Making the change later would put the next work into backlog most likely. Better to do it now.